arctic/store/version_store.py

```diff
@@ -106,6 +106,16 @@ def __str__(self):
     def __repr__(self):
         return str(self)
     
+    def _get_data_info(self, version):
```

Comment:
Delegate this down to the _arctic_lib (in our instance PandasNDArrayStore) - that way we can implement different implementations for multiple types.